File: src/AHF-API/DataLayer/Model/Inventory/InventoryItem.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DataLayer.Model.Inventory
{
    public abstract class InventoryItem
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int Quantity { get; set; }
    }
}

File: src/AHF-App/app/src/main/java/com/ahf/antwerphasfallen/InventoryItem.java
package com.ahf.antwerphasfallen;

/**
 * Created by Jorren on 16/11/2018.
 */

public abstract class InventoryItem {
    private String name;
    private int quantity;

    public InventoryItem() {
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getQuantity() {
        return quantity;
    }

    public void setQuantity(int quantity) {
        this.quantity = quantity;
    }
}
